src/segmantic/seg/transforms.py

```diff
@@ -0,0 +1,119 @@
+import warnings
+from pathlib import Path
+from typing import Dict, Optional, Sequence, Union
+
+import torch
+import yaml
+from monai.config.type_definitions import KeysCollection, NdarrayOrTensor
+from monai.networks import one_hot
+from monai.transforms.post.array import Ensemble
+from monai.transforms.post.dictionary import Ensembled
+from monai.transforms.transform import Transform
+from monai.utils import TransformBackends
+
+
+class SelectBestEnsemble(Ensemble, Transform):
+    """
+    Execute vote ensemble on the input data.
+    The input data can be a list or tuple of PyTorch Tensor with shape: [C[, H, W, D]],
+    Or a single PyTorch Tensor with shape: [E[, C, H, W, D]], the `E` dimension represents
+    the output data from different models.
+    Typically, the input data is model output of segmentation task or classification task.
+
+    Note:
+        This vote transform expects the input data is discrete values. It can be multiple channels
+        data in One-Hot format or single channel data. It will vote to select the most common data
+        between items.
+        The output data has the same shape as every item of the input data.
+
+    Args:
+        num_classes: if the input is single channel data instead of One-Hot, we can't get class number
+            from channel, need to explicitly specify the number of classes to vote.
+
+    """
+
+    backend = [TransformBackends.TORCH]
+
+    def __init__(
+        self,
+        candidate_per_tissue_path: Path,
+        tissue_dict: Dict[str, int],
+        num_classes: Optional[int] = None,
+    ) -> None:
+
+        self.num_classes = num_classes
+        self.tissue_dict = tissue_dict
+        with open(f"{candidate_per_tissue_path}") as file:
+            self.best_candidate_for_tissue = yaml.full_load(file)
```

Comment:
always use `yml.safe_load`, see e.g. https://stackoverflow.com/a/63916315/8291595

Reply:
but consider using [`segmantic.utils.config.load`](https://github.com/dyollb/segmantic/blob/34a3e399e558267f084c03d7e8d6679632c16a23/src/segmantic/utils/config.py#L9)

Reply:
ahh thats good to know. thank you. I used the config.load one.